app/views/articles/show.html.erb

```diff
@@ -13,5 +13,23 @@
   <%= @article.status %>
 </p>
 
+<% if @article.status == 'draft' %>
```

Comment:
You should do it with policy, because this realisation on view level only. Unauthorized user still can send data to contoller.